scheme/app/DoctrineMigrations/Version20190408170724.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20190408170724 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE Friends ADD interCompanyId INT UNSIGNED DEFAULT NULL, CHANGE directConnectivity directConnectivity VARCHAR(255) DEFAULT \'yes\' NOT NULL COMMENT \'[enum:yes|no|intervpbx]\'');
```

Comment:
255 chars seems a bit excessive for a field with less than 10 possible characters.